spec/mini_twitter_client_spec.rb

```diff
@@ -0,0 +1,24 @@
+require './lib/mini_twitter_client'
+
+RSpec.describe 'methods of mini_twitter_client class: create_tweet(author, massage) and delete_tweet(id)' do
+  it ' creates new tweet and delete it ' do
+    author = 'Johan'
+    message = 'Test tweet :)'
+
+    def mini_twitter_client
+      MiniTwitterClient.new
+    end
+
+    res = []
+    response = mini_twitter_client.create_tweet(author, message)
+    res << 200 if [200, 201, 204].include?(response.status)
+
+    id = response.headers[:location].split("/")[-1]
```

Comment:
It would be better to take tweet id from response body. But it can be done inside `mini_twitter_client.create_tweet` method.

`mini_twitter_client.create_tweet` could return created tweet, and you could just take the id from it:

```ruby
tweet[:id/]
```